chains/orchestrator-relays/runtime/starlight/src/lib.rs

```diff
@@ -0,0 +1,3813 @@
+}
+
+pub struct LeafExtraDataProvider;
+impl BeefyDataProvider<LeafExtraData> for LeafExtraDataProvider {
+    fn extra_data() -> LeafExtraData {
+        let mut para_heads: Vec<(u32, Vec<u8>)> = parachains_paras::Parachains::<Runtime>::get()
+            .into_iter()
+            .filter_map(|id| {
+                parachains_paras::Heads::<Runtime>::get(id).map(|head| (id.into(), head.0))
+            })
+            .collect();
+        para_heads.sort();
+        let para_heads_root = binary_merkle_tree::merkle_root::<mmr::Hashing, _>(
+            para_heads.into_iter().map(|pair| pair.encode()),
+        );
+
+        let commitment_root =
+            OutboundMessageCommitmentRecorder::take_commitment_root().unwrap_or_default();
+
+        LeafExtraData {
+            para_heads_root,
+            commitment_root,
+        }
+    }
+}
+
+impl pallet_beefy_mmr::Config for Runtime {
+    type LeafVersion = LeafVersion;
+    type BeefyAuthorityToMerkleLeaf = pallet_beefy_mmr::BeefyEcdsaToEthereum;
+    type LeafExtra = LeafExtraData;
+    type BeefyDataProvider = LeafExtraDataProvider;
+    type WeightInfo = weights::pallet_beefy_mmr::SubstrateWeight<Runtime>;
+}
+
+impl paras_sudo_wrapper::Config for Runtime {}
+
+use {
+    pallet_pooled_staking::traits::{IsCandidateEligible, Timer},
+    pallet_staking::SessionInterface,
+};
+
+pub struct StarlightSessionInterface;
+impl SessionInterface<AccountId> for StarlightSessionInterface {
+    fn disable_validator(validator_index: u32) -> bool {
+        Session::disable_index(validator_index)
+    }
+
+    fn validators() -> Vec<AccountId> {
+        Session::validators()
+    }
+
+    fn prune_historical_up_to(up_to: SessionIndex) {
+        Historical::prune_up_to(up_to);
+    }
+}
+
+prod_or_fast_parameter_types! {
+    pub const SessionsPerEra: SessionIndex = { prod: 4, fast: 3 };
+    pub const SlashDeferDuration: EraIndex = { prod: 0, fast: 0 };
+}
+
+impl pallet_external_validators::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type UpdateOrigin = EnsureRoot<AccountId>;
+    type HistoryDepth = ConstU32<84>;
+    type MaxWhitelistedValidators = MaxWhitelistedValidators;
+    type MaxExternalValidators = MaxExternalValidators;
+    type ValidatorId = AccountId;
+    type ValidatorIdOf = ValidatorIdOf;
+    type ValidatorRegistration = Session;
+    type UnixTime = Timestamp;
+    type SessionsPerEra = SessionsPerEra;
+    type OnEraStart = (ExternalValidatorSlashes, ExternalValidatorsRewards);
+    type OnEraEnd = ExternalValidatorsRewards;
+    type WeightInfo = weights::pallet_external_validators::SubstrateWeight<Runtime>;
+    #[cfg(feature = "runtime-benchmarks")]
+    type Currency = Balances;
+}
+
+pub struct TimestampProvider;
+impl Get<u64> for TimestampProvider {
+    fn get() -> u64 {
+        Timestamp::get()
+    }
+}
+
+parameter_types! {
+    // Chain ID of Sepolia.
+    // Output is: 34cdd3f84040fb44d70e83b892797846a8c0a556ce08cd470bf6d4cf7b94ff77
```

Comment:
this output  and comment are probably wrong after we change the chain id

Reply:
It's incorrect even for the Dancelight, updated for both: Starlight/Dancelight.